backend/validators/models.py

```
@@ -111,6 +111,45 @@ def __str__(self):
+
+def ensure_validator_profile(user):
+    user_id = getattr(user, 'id', None)
+    if not user_id:
+        return None
+
+    validator, _ = Validator.objects.get_or_create(user_id=user_id)
+
+    try:
+        user._state.fields_cache['validator'] = validator
+    except Exception:
+        pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)